configs/components/oasis3mct/oasis3mct.env.yaml

```diff
@@ -4,9 +4,22 @@ add_compiletime_environment_changes:
                         add_export_vars:
                                 - 'OASIS_FFLAGS=-emf'
         choose_computer.name:
+                juwels:
+                        add_export_vars:
+                                OASIS_FFLAGS: '"-fPIC"'
+                                OASIS_CFLAGS: '"-fPIC"'
+                                CFLAGS: '"-fPIC"'
+                                CCFLAGS: '"-fPIC"'
+                                FFLAGS: '"-fPIC"'
+                                FCFLAGS: '"-fPIC"'
                 levante:
                         add_export_vars:
-                                OASIS_FFLAGS: '"-march=core-avx2 -mtune=core-avx2"'
+                                OASIS_FFLAGS: '"-fPIC"'
+                                OASIS_CFLAGS: '"-fPIC"'
+                                CFLAGS: '"-fPIC"'
+                                CCFLAGS: '"-fPIC"'
+                                FFLAGS: '"-fPIC"'
+                                FCFLAGS: '"-fPIC"'
```

Comment:
This is introducing changes in all models using Oasis, what does this change exactly do?

Reply:
ah, I see, and it is supported in Levante and Juwels, and that's why we are seeing this here right? Thanks for the info :)